spec/features/navigation/regular_user_navigation_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe "Registered User" do
+  describe "User's View" do
+    before(:each) do
+      @user = User.create(name: "Dee",
+                         street_address: "4233 Street",
+                         city: "Golden",
+                         state: "CO",
+                         zip: "80042",
+                         email: "deedee@gmail.com",
+                         password: "rainbows1908",
+                         role: 0)
+
+      # session[:user_id] = @user.id
```

Comment:
Can we delete this now that we have completed this test?

Reply:
Yes!